src/pcrlock/pcrlock.c

```diff
@@ -2691,6 +2691,95 @@ static int make_pcrlock_record(
         return 0;
 }
 
+static void evp_md_ctx_free_all(EVP_MD_CTX *(*md)[TPM2_N_HASH_ALGORITHMS]) {
+        assert(md);
+        for (size_t i = 0; i < sizeof(*md) / sizeof(**md); i++)
+                if ((*md)[i])
+                        EVP_MD_CTX_free((*md)[i]);
+}
+
+static int make_pcrlock_record_from_stream(
+                uint32_t pcr,
+                FILE *f,
+                JsonVariant **ret_record) {
+
+        _cleanup_(json_variant_unrefp) JsonVariant *digests = NULL;
+        _cleanup_(evp_md_ctx_free_all) EVP_MD_CTX *mdctx[TPM2_N_HASH_ALGORITHMS] = { NULL };
```

Comment:
drop the `NULL`. In our codebase you can just write `{}` here. It's a GNU extension we happily use.

Reply:
I see. Done!